Formula/create-api.rb

```diff
@@ -0,0 +1,29 @@
+class CreateApi < Formula
+  desc "Delightful code generator for OpenAPI specs"
+  homepage "https://github.com/CreateAPI/CreateAPI"
+  url "https://github.com/CreateAPI/CreateAPI/archive/refs/tags/0.0.5.tar.gz"
+  sha256 "c250ff140af83e093d86fef0dd18b87363ae91a087c0804bedb40f9922989093"
+  license "MIT"
+  head "https://github.com/CreateAPI/CreateAPI.git", branch: "main"
+
+  depends_on xcode: ["13.0", :build]
+
+  uses_from_macos "swift"
+
+  resource "homebrew-test-data" do
+    url "https://github.com/CreateAPI/CreateAPI/archive/refs/tags/0.0.5.tar.gz"
+    sha256 "c250ff140af83e093d86fef0dd18b87363ae91a087c0804bedb40f9922989093"
+  end
+
+  def install
+    system "swift", "build", "--disable-sandbox", "--configuration", "release"
+    bin.install ".build/release/create-api"
+  end
+
+  test do
+    resource("homebrew-test-data").stage testpath
+
+    schema = testpath/"Tests/CreateAPITests/Specs/github.yaml"
+    system bin/"create-api", "generate", "--package", "GitHub", schema
```

Comment:
Can we check that it generated something? Or maybe check the output of this command? (preferably the former)

Reply:
Ah yes we could indeed. I can actually test that the generated package compiles. Would that be a good option? Or would you prefer to check that some given files actually exist?

Reply:
Checking that it compiles seems more useful 👍